<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>676</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>676</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>韩都衣舍活动管理图片筛选器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>F:/workspace/mdm/web/target/mdm/main/webapp/static/img/favicon/favicon.ico</normaloff>F:/workspace/mdm/web/target/mdm/main/webapp/static/img/favicon/favicon.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>3.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/back/D:/MainForm.jpg)</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/back/MainForm.jpg)</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>检测结果</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/back/MainForm.jpg)</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>检测图片</string>
    </attribute>
    <widget class="QPushButton" name="check">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>检测图片</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>文件目录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>请选择图片目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file_2">
      <property name="text">
       <string>选择文件夹</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>挑选图片</string>
    </attribute>
    <widget class="QPushButton" name="pick">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>图片挑选</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>文件目录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>请选择保存文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="desfile">
      <property name="text">
       <string>选择文件夹</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>选择Excel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>请选择货号文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excelfile">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件目录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>请选择图片文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file">
      <property name="text">
       <string>选择文件夹</string>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="copy">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>复制</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="move">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>移动</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gana项目组制作 ，请给红心哈 ^_^！</string>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources>
  <include location="D:/back.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
